### PR TITLE
[Feature/51601] Adjusted flow for denying countries consent

### PR DESCRIPTION
**Pull request by jonasofrich**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/CountriesConsentActivity.cs [hunk 1]
```diff
@@ -67,19 +66,10 @@ private void InitLayout()
             _closeButton.ContentDescription = CLOSE_BUTTON_ACCESSIBILITY_LABEL;
 
             //Button click events
-            _closeButton.Click += new StressUtils.SingleClick((sender, e) => ShowAreYouSureToExitDialog(), 500).Run;
-            _consentButton.Click += new StressUtils.SingleClick(async (o, args) =>
+            _closeButton.Click += new StressUtils.SingleClick((sender, e) => ShowAreYouSureToDenyConsentDialog(), 500).Run;
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/CountriesConsentActivity.cs:69]: i do not understand.. is this really requested change? this is totally not user friendly. close button is always understood as leaving the whole flow, and now user will be redirected to the next page? @Frederik-S-N-dk

Comment (Frederik-S-N-dk) [reply NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/CountriesConsentActivity.cs:69]: @nc-kano I'll ask FHI again. But at least that is what we agreed upon on the last meeting.
So wait with merge of this PR until I have clarified with them again

Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/CountriesConsentActivity.cs:69]: @Frederik-S-N-dk ok, user must be aware that this is totally not user friendly and misleading to the user. as i remember there is also a requirement in Android review process that we should not change the default button behaviors such as back/close/next/arrow buttons.
image

**Comment by Frederik-S-N-dk** [comment]
New user flow has been defined. Thus, this PR is closed.

**Pull request closed without merging**